### DISCUSSION TITLE
NOUS A5T - relay don't turn off the power anymore

### DISCUSSION THREAD

**Discussion by Filiaes** [arendst/Tasmota#21650] [Q&A] [answered]
Hi, I am not sure if this is the right place to ask for that problem. If not direct me into the right direction please. I have a NOUS A5T and it is working since around two years without any problem. I noticed that one of the three relais?/input slots don't switch the power mode for the device attached to it anymore. On the web interface I can toogle the slot and is is switching between on and off but the device attached to that slot just stays online and doesn't turn off.  Also the status light for that slot is turning on and off and I can see in the console that the Power is switched on and off - so from software perspective everything is fine but the hardware doesn't change the status of the device anymore.

Is this common? Can I do something to fix that?

Thanks in advance
Fil

**Comment by barbudor** [comment, marked as answer]
If the template has not been changed and is still correct, there's a high chance for a hardware failure.
If it's still under warranty, you may have to contact the seller.

**Reply by sfromis** [reply]
> Also the status light for that slot is turning on and off and I can see in the console that the Power is switched on and off

That does suggest that Tasmota is doing what it should, as per-plug status leds are not separately controlled by Tasmota, but tied to the signal given to the relays. So yes, hardware failure does sound like the answer.
